### DISCUSSION TITLE
[Enhancement]: Restart button for updates

### DISCUSSION THREAD

**Discussion by Aria-NL** [ciderapp/Cider#625] [Feature Request]
### The feature you would like

Currently, the user is required to manually close and reopen Cider after any update. Additionally, Cider does not provide any feedback to the user saying that the update was complete after closing the application. It would be more clear for the user if there was a "Restart" button that appeared when an update has been downloaded. See: the update ready button in Discord

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by coredev-uk** [comment]
Try build from [3703007](https://github.com/ciderapp/Cider/commit/37030073d8d914d94988d287a4bec354bf3e8189)
